examples/example-jwt-auth/README.md

```diff
@@ -0,0 +1,65 @@
+Authentication Example
+==============================================
+
+This example illustrates a simple JWT-based authentication implementation in gRPC using
+ server interceptor. It uses the JJWT library to create and verify JSON Web Tokens (JWTs).
+
+The example requires grpc-java to be pre-built. Using a release tag will download the relevant binaries
+from a maven repository. But if you need the latest SNAPSHOT binaries you will need to follow
+[COMPILING](../../COMPILING.md) to build these.
+
+The source code is [here](src/main/java/io/grpc/examples/jwtauth). Please follow the
+[steps](../README.md#-to-build-the-examples) to build the examples. The build creates scripts
```

Comment:
This does not seem to do it - have you verified these steps? 
I had to execute
```
../gradlew installDist
```
in the current directory.

If your structure is similar to `example-tls` shouldn't your instructions also match?

Reply:
Yes, to execute from the current directory you need to use `../gradlew installDist`, but instruction says about building all examples and references file in another directory.
This is consistent with the `example-gauth`.

Reply:
Anyway I will supplement the instructions for building only this example.